### PR TITLE
Feature/1326 logo iconcomponent

### LINKED ISSUE #1326: Put AXA Logo to icon-component
An icon speaks for itself. The axa logo does not fit into this category.

A button with a download icons says "if you click me, you download something".
A button with a axa logo says nothing. It doesn't speak for itself.

The axa logo should be only under /images.

### PR DESCRIPTION
**Pull request by danyball**
Fixes #1326

# Done is when (DoD):
- [x] I have added UI/Unit tests for my changes
- [ ] Design has been reviewed with (here name of reviewer)
- [x] Old tests and eslint rule are not broken
- [x] My changes are well documented in the readme and showcased in the stories 

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas (focus on **WHY**)
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by danyball** [comment]
Die 2 Logos haben unterschiedliche Grössen in `viewBox`. Habe mit Markus gesprochen. Er meinte das ist ok, weil der Verwender die Grösse so oder so selbst einstellen muss. 

- SVG unter /images - 96x96
- axa-logo - 283x283
- axa-logo-open - 80x80

Ok so?

File: src/components/10-atoms/icon/README.md [hunk 1]
```diff
@@ -49,24 +53,30 @@ Import the icon-defining script and use icons like so:
...
-| `icon="email"`       |
-| `icon="expand"`      |
-| `icon="mobile"`      |
-| `icon="phone"`       |
-| `icon="plus"`        |
-| `icon="search"`      |
-| `icon="upload"`      |
+| Attribute              |
+| ---------------------- |
+| `icon="arrow-right"`   |
```
Comment (01231) [inline src/components/10-atoms/icon/README.md:59]: add `arrow-left`

File: src/components/10-atoms/icon/story.js [hunk 2]
```diff
@@ -14,40 +20,75 @@ storyIcon.addParameters({
...
-  const template = html`
-    <axa-icon icon="${icon}"></axa-icon>
-  `;
+    const loadIconType = radios(
+      'Load icon this way:',
+      { 'Named property': 'prop', URL: 'url', 'SVG string': 'string' },
+      'prop'
+    );
+    const icon = select('Value for "Named property"', iconList, 'download');
+    const itemIconUrl = text('Value for "URL"', '/svg/logo-axa.svg');
```
Comment (01231) [inline src/components/10-atoms/icon/story.js]: I get this error when I try to use the option `Value for "URL"`
Screenshot 2019-10-17 at 14 51 01

File: src/components/10-atoms/icon/story.js [hunk 3]
```diff
@@ -14,40 +20,75 @@ storyIcon.addParameters({
...
+    const itemIconString = text(
+      'Value for "SVG string"',
+      '<svg width="100px" height="100px" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 96 96"><g fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"><path d="M34.11 74.27v4.6m34.5-48.23A21.21 21.21 0 0 0 48 15.5a21.23 21.23 0 0 0-20.53 15.07m.43 12.23a21.61 21.61 0 0 0 40.29-.07m1.19-12.18a4.91 4.91 0 0 0-.77.09 19.36 19.36 0 0 1-.42 12.09 5.23 5.23 0 0 0 1.19.14A5.92 5.92 0 0 0 75 36.71a5.92 5.92 0 0 0-5.62-6.16zm-1.19 12.18a19.36 19.36 0 0 0 .42-12.09"/><path d="M27.31 42.87a4 4 0 0 0 .59-.07 19.34 19.34 0 0 1-.43-12.23h-.16m29.51-8.23c-2.77 0-5-3.45-5-6.47m2.49 10.58c-4.85 0-8.79-5.29-8.79-10.58M24.33 76.94l12.84 2.53a61.28 61.28 0 0 0 22.35 0l12.84-2.53m-9.78-2.67v4.6"/><path d="M24.33 76.94c0-13.26 10.75-20.46 24-20.46s24 7.2 24 20.46M54.57 57.37a6.23 6.23 0 1 1-12.45 0M27.81 42.73a5.23 5.23 0 0 1-1.19.14A5.92 5.92 0 0 1 21 36.71a5.92 5.92 0 0 1 5.65-6.16 4.91 4.91 0 0 1 .77.09"/></g></svg>'
+    );
 
-  render(template, wrapper);
-  return wrapper;
-});
+    const createListWithAllIcons = () => {
+      const list = document.createElement('ul');
```
Comment (01231) [inline src/components/10-atoms/icon/story.js:42]: You could us html template for creating the list

**Review by 01231 — CHANGES_REQUESTED** [review]

File: src/components/00-materials/README.md [hunk 4]
```diff
@@ -74,20 +74,24 @@ this.render() {
...
-    - b. alternatively use the [online SVG optimizer](https://jakearchibald.github.io/svgomg/] for the same purpose
+   - a. `data-name="Layer 2"` # layer info for vector drawing software
+   - b. `<path fill="#fff" d="M0 0h96v96H0z"/>` # outer bounding box filled with white background
+3. - a. use the internal [svgo](https://github.com/svg/svgo)-invoking scripts `npm run build-icons` or `npm run build-images` to clean up and optimize the SVGs, then copy the optimized files back from the `.tmp` folder to `icons-raw` or `images-raw`
+   - b. alternatively use the [online SVG optimizer](https://jakearchibald.github.io/svgomg/] for the same purpose
 4. Manually review the SVGs produced in the last step: add or edit &lt;path&gt; attributes, setting `fill="currentColor"` and/or `strokes="currentColor"` as appropriate. Remove unnecessary attributes.
+
+## ChangeLog
+- 2.0.0
+    - removed Axa Logos from /icons
```
Comment (raphaellueckl) [inline src/components/00-materials/README.md]: And you moved it to images, right? Or was it duplicated all along?

File: src/components/10-atoms/icon/README.md [hunk 5]
```diff
@@ -75,21 +85,7 @@ The default behavior set the size of the svgs to 16x16 px, to use original size
 | ------------- |
 | `size="auto"` |
 
-## Using custom icons
```
Comment (raphaellueckl) [inline src/components/10-atoms/icon/README.md:78]: Why did you remove this part?

File: src/components/10-atoms/icon/index.js [hunk 6]
```diff
@@ -67,21 +68,20 @@ class AXAIcon extends LitElement {
...
-      req.send();
+      xhrCall(icon).then(result => {
+        this.size = 'auto';
+        this._loadedSvg = result;
+      });
+    } else if (/<svg/.test(icon)) {
+      this.size = 'auto';
+      this._loadedSvg = icon;
+    } else {
+      this._loadedSvg = AXAIcon.iconsMapping[icon] || '';
```
Comment (raphaellueckl) [inline src/components/10-atoms/icon/index.js:79]: Doesn't this create a boolean expression instead of a string?

**Review by raphaellueckl — CHANGES_REQUESTED** [review]

File: src/components/10-atoms/icon/story.js (hunk 3 above)
Comment (danyball) [reply src/components/10-atoms/icon/story.js:42]: Not my code - it works and its ok for me ;)

File: src/components/00-materials/README.md (hunk 4 above)
Comment (danyball) [reply src/components/00-materials/README.md]: It was a duplicate. Just some technical differences. We have to release materials as a breaking change because of that.

File: src/components/10-atoms/icon/README.md (hunk 5 above)
Comment (danyball) [reply src/components/10-atoms/icon/README.md:78]: That information is not relevant for the icon component. Just the info that you have to set the size of the SVG is relevant for my opinion. And i moved that info to another place in the readme

File: src/components/10-atoms/icon/index.js (hunk 6 above)
Comment (danyball) [reply src/components/10-atoms/icon/index.js:79]: No, read it that way:

If `AXAIcon.iconsMapping[icon] ` is falsy, take the second part of OR. Which is empty string.

File: src/components/10-atoms/icon/story.js (hunk 2 above)
Comment (danyball) [reply src/components/10-atoms/icon/story.js]: This is a `feature branch deployment` bug. I will create an issue.

**Comment by raphaellueckl** [comment]
The knobs are broken :/

File: src/components/10-atoms/icon/story.js (hunk 2 above)
Comment (danyball) [reply src/components/10-atoms/icon/story.js]: Found a solution. Should work now even on feature branch deploy

**Comment by danyball** [comment]
> The knobs are broken :/

Re-Run Travis and the build is running again?! :)

File: src/components/00-materials/README.md (hunk 4 above)
Comment (raphaellueckl) [reply src/components/00-materials/README.md]: Didn't you want to write `- Removed AXA logos from /icons and moved them to /images`?

**Review by raphaellueckl — CHANGES_REQUESTED** [review]

File: src/components/00-materials/README.md (hunk 4 above)
Comment (danyball) [reply src/components/00-materials/README.md]: Jep, pushed that right now
